CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.18.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.18.0-rc1) - 2022-11-15
+## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/1.18.0) - 2022-12-22
 
+* SECURITY
+  * Support Go Vulnerability Management (#21139)
+  * Forbid HTML string tooltips (#20935)
 * BREAKING
   * Remove U2F support (#20141)
```

Comment:
```suggestion
  * Rework mailer settings (#18982)
  * Remove U2F support (#20141)
  * Refactor `i18n` to `locale` (#20153)
  * Enable contenthash in filename for dynamic assets (#20813)
```

See [breaking changes](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.18.0+is%3Amerged+label%3Akind%2Fbreaking).

Reply:
I know that, but looks like previous rc0 and rc1 changelog move them into other classes.

Reply:
IMHO the fact these are breaking changes is the most important.